### ISSUE TITLE
greedy_layerwise_train in dbn.py just print a line?

### ISSUE THREAD

**Issue by thinkinds** [wuaalb/keras_extensions#16]
other sentences in this function have been commented by a pair of triple quotes 

> def greedy_layerwise_train(self,layer_no, X, batch_size=128, nb_epoch=100, verbose=1, shuffle="batch"):

        print ">>>",layer_no
        '''
        if layer_no !=0:
            pre_model = Sequential()
            for i,rbm in enumerate(self.rbms[0:layer_no]):
                pre_model.add(rbm.get_h_given_x_layer((i==0)))
                pre_model.add(SampleBernoulli(mode='random'))
            pre_model.compile(SGD(),loss='mean_squared_error')
        '''
